zerver/openapi/zulip.yaml

```diff
@@ -15910,6 +15910,19 @@ paths:
             type: boolean
           example: false
           required: false
+        - name: is_default_stream
+          in: query
+          description: |
+            Add or remove the stream as a [default stream][default-stream]
+            for new users joining the organization.
+
+            [default-stream]: /help/set-default-streams-for-new-users
+
+            **Changes**: New in Zulip 8.0 (feature level 193).
```

Comment:
We can add "Previously, default stream status could only be toggled using the [dedicated API endpoint](/api/add-default-stream).`